### PR TITLE
Show case number on emancipation checklist

### LINKED ISSUE #1610: Show case number on emancipation checklist
**What type of user is this for? volunteer/supervisor/admin/all**
all

**Description**

emancipation checklist should reference case - make it easier for volunteers w multiple youth, and for supervisors and admins

**Screenshots of current behavior, if any**
show-case

_QA Login Details:_
url: https://casa-qa.herokuapp.com/
password for all users: 123456

Log in using any of the below emails, depending on the account type you'd like to log in as.
- volunteer1@example.com
- supervisor1@example.com
- casa_admin1@example.com

### PR DESCRIPTION
**Pull request by bmanek**
### What github issue is this PR for, if any?
Resolves #1610 

### What changed, and why?
Added link per issue description. Test coverage for link/feature is nearly complete, but unsure how to get it to 100%.

### How will this affect user permissions?
- Volunteer permissions:
- Supervisor permissions:
- Admin permissions:

### How is this tested? (please write tests!) 💖💪
Test is mostly complete

### Feelings gif (optional)
What gif best describes your feeling working on this issue? https://giphy.com/
How to embed:
`alt text`

### CODE REVIEW

File: spec/views/emancipations/show.html.erb_spec.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  let(:admin) { create(:casa_admin, casa_org: organization) }
+  let(:casa_case) { create(:casa_case) }
+  let(:emancipation_form_data) { [create(:emancipation_category)] }
+
+
+  before do
+    assign :current_case, casa_case
+    assign :emancipation_form_data, emancipation_form_data
+
+    # enable_pundit(view, user)
```
Comment (compwron) [inline spec/views/emancipations/show.html.erb_spec.rb:16]: Let's remove this

File: spec/views/emancipations/show.html.erb_spec.rb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    # allow(view).to receive(:current_user).and_return(user)
+    # allow(view).to receive(:current_organization).and_return(user.casa_org)
+  end
+
+  it "has a link to return to case from emancipation" do
+    sign_in admin
+    render template: "emancipations/show"
+
+    # old:
```
Comment (compwron) [inline spec/views/emancipations/show.html.erb_spec.rb:25]: remove comment

**Comment by compwron** [comment]
Looks good!
I'm happy to merge this as is. To see how to increase test coverage, open your local file `coverage/index.html` click on the file you want to view, and look at the red-highlighted lines (if any). Then write a test to exercise the red lines. 
Here are some resources, and also I'd love to talk this over during office hours tomorrow :) 
https://www.toptal.com/qa/how-to-write-testable-code-and-why-it-matters
https://softwareengineering.stackexchange.com/questions/120775/how-to-drastically-improve-code-coverage
https://www.simform.com/test-coverage/

**Comment by bmanek** [comment]
Sounds great, thanks! I'll read the article tonight and see how far I get
in writing coverage.

**Comment by compwron** [comment]
I hope to merge this during office hours tonight

**Comment by compwron** [comment]
I want to get this into the Friday deploy so I'm going to merge it and then clean it up in situ. Thanks for the work!! :) 💖💪

**Pull request merged**